Source/DevHelper/CodeGenerators/LinkedPathCreationWidgetCodeGenerator.cpp

```diff
@@ -0,0 +1,92 @@
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE
+ * USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ * The code contained herein was partially funded by the followig contracts:
+ *    United States Air Force Prime Contract FA8650-07-D-5800
+ *    United States Air Force Prime Contract FA8650-10-D-5210
+ *    United States Prime Contract Navy N00173-07-C-2068
+ *
+ * ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ */
+
+#include "LinkedPathCreationWidgetCodeGenerator.h"
+
+// -----------------------------------------------------------------------------
+//
+// -----------------------------------------------------------------------------
+LinkedPathCreationWidgetCodeGenerator::LinkedPathCreationWidgetCodeGenerator(const QString& humanLabel, const QString& propertyName, const QString& category, const QString& initValue)
+: FPCodeGenerator(humanLabel, propertyName, category, initValue, "DataArrayPath")
+{
+}
+
+// -----------------------------------------------------------------------------
+//
+// -----------------------------------------------------------------------------
+LinkedPathCreationWidgetCodeGenerator::~LinkedPathCreationWidgetCodeGenerator() = default;
+
+// -----------------------------------------------------------------------------
+//
+// -----------------------------------------------------------------------------
+QString LinkedPathCreationWidgetCodeGenerator::generateSetupFilterParameters()
+{
+  QString s;
+  QTextStream out(&s);
+  //out << "  parameters.push_back(SIMPL_NEW_AM_WITH_LINKED_DC_FP(\"" << getHumanLabel() << "\", " << getPropertyName() << ", " << getCategory() << ", @FilterName@));";
+  out << "#error The correct code for the LinkedPathCreation Filter Parameter needs to be inserted here. Please see the documentation for the filter parameter.";
+  return s;
+}
+
+// -----------------------------------------------------------------------------
+//
+// -----------------------------------------------------------------------------
+QString LinkedPathCreationWidgetCodeGenerator::generateDataCheck()
+{
+  return "";
+}
+
+// -----------------------------------------------------------------------------
+//
+// -----------------------------------------------------------------------------
+QString LinkedPathCreationWidgetCodeGenerator::generateFilterParameters()
+{
+  QString contents;
+  QTextStream ss(&contents);
+  ss << "    SIMPL_FILTER_PARAMETER(DataArrayPath, " + getPropertyName() + ")\n";
+  ss << "    Q_PROPERTY(DataArrayPath " + getPropertyName() + " READ get" + getPropertyName() + " WRITE set" + getPropertyName() + ")";
```

Comment:
Change this to a QString property and update the copyright header to match the .h file, and this should be good.